core/src/main/resources/hudson/PluginManager/_installed.js

```diff
@@ -0,0 +1,55 @@
+(function () {
+  const restartRequired = document
+    .querySelector("#is-restart-required-for-completion")
```

Comment:
[nit] It does not matter one way or another, but `getElementById` is more common in this codebase to select an element by its ID (110 usages as opposed to 29 usages of `querySelector("#`). It also performs a little better in theory, but not enough to matter in practice. No strong preference one way or another, so feel free to ignore this comment if you want.

```suggestion
    .getElementById("is-restart-required-for-completion")
```